docs/console/usage-billing.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: usage-billing
+title: Usage-based billing
+sidebar_label: Usage-based billing
+slug: usage-billing
+---
+
+# Usage-based billing
+
+:::info
+
+This feature is in preview. While we are rolling out usage-based billing, we will not collect payment for overusage. Payment
+collection will start XX.XX.2024.
+
+Please report any issues to support@ory.sh or contact us in the [Ory Community](https://slack.ory.sh/).
+
+:::
+
+## Overview
+
+Usage-based invoices is a new feature in the Ory Console that you to understand your usage of Ory Cloud services, the usage that
+is included in your plan, and the overuse. Usage-based invoices are generated on a monthly basis.
+
+## Enabling Usage-based Invoices
+
+Displaying the usage-based invoices is guarded by a feature toggle and is disabled by default. To enable it, navigate to the
+[preview features page](https://console.ory.sh/projects/current/features) and enable the `Show invoices` feature. The Ory Console
```

Comment:
TODO, once my rework is merged, it's https://console.ory.sh/settings/advanced and the page won't reload anymore.

Reply:
done!